frontend/src/metabase/plugins/index.ts

```diff
@@ -54,6 +54,12 @@ export const PLUGIN_ADMIN_TOOLS = {
 export const PLUGIN_ADMIN_SETTINGS_UPDATES = [];
 
 // admin permissions
+export const PLUGIN_ADMIN_PERMISSIONS_DATABASE_ROUTES = [];
+export const PLUGIN_ADMIN_PERMISSIONS_DATABASE_GROUP_ROUTES = [];
+export const PLUGIN_ADMIN_PERMISSIONS_DATABASE_POST_ACTION = {
+  impersonated: null,
+};
+
```

Comment:
is the reason for these plugins to keep the EE code in a separate directory?

Reply:
Yes, this file is like the interface for EE code which lets us have zero direct imports of any enterprise code